transforms/language/text_encoder/python/README.md

```diff
@@ -0,0 +1,47 @@
+# Text Encoder Transform 
+
+## Summary 
+This transform is using sentence encoder models to create embeddings of the text in each row of the input .parquet table.
```

Comment:
Can you provide some reference or documentation for what an `embedding` is and how it might be used downstream?  Yes, I'm still learning, but others might be as well :).